### PR TITLE
dayView and monthView updates

### PR DESCRIPTION
**Pull request by lxu03**
dayView: events align with the minutes they start and end, and hitboxes for events are slightly better
monthView: now has full functionality, but displays event cards instead of a copy of dayView

### CODE REVIEW

File: lib/view/monthlyEventView.dart [hunk 1]
```diff
@@ -0,0 +1,128 @@
+import 'package:flutter/material.dart';
```
Comment (stxue1) [inline lib/view/monthlyEventView.dart:1]: Is this just the old monthlyeventview?

File: lib/view/dailyEventView.dart [hunk 2]
```diff
@@ -153,19 +143,19 @@ class _SingleDayState extends State<SingleDay> {
...
-                    width: space, child: const Card(color: Colors.black))),
+          (event) => GestureDetector(
+            onTap: () {
+              showEventDetailPopup(context, event, widget.date);
+            },
+            child: CustomPaint(
+              painter: MyPainter(context, eventWidth: space, event: event),
+              child: Container(
+                width: space,
+                height: hourHeight,
```
Comment (stxue1) [inline lib/view/dailyEventView.dart:156]: I think GestureDetector expands to fit all of its children. Maybe [something like this](https://stackoverflow.com/a/74403333) could be done as quick fix before the release?

File: lib/view/monthlyEventView.dart (hunk 1 above)
Comment (lxu03) [reply lib/view/monthlyEventView.dart:1]: yep, the old one was misspelled "monthlyEvenView"

**Comment by Baron-Paelen** [comment]
On first launching the app, event daily view does not show all events. You need to force a refresh by going into a different view and then coming back. I don't think this should be merged in until this is solved.

**Review by stxue1 — APPROVED** [review]
llgtm

**Review by anlhu — COMMENTED** [review]
code looks ok

**Pull request merged**